README explains patterns DSL (#72)
mjackson/remix-the-web: packages/route-pattern/README.md

> # URL Patterns

Reviewer: ```suggestion
# Route Pattern
```

> URL patterns are strings that describe the structure of URLs you want to match.

Reviewer: ```suggestion
Route patterns are strings that describe a URL.
```
